vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,125 @@
+
+### Entando Identity Management System 
+Logging into the Entando Identity Management System, you will see the landing page below. The left navigation bar is your guide for managing users, groups, and most importantly, roles for your application. Using the RBAC model, you define what users have access to by the roles and groups they are assigned to. Some important guidelines are noted below. 
+
+![Entando Identity Management System](./images/custportal-idmanagement-main.png)
+
+A. **Define the Realm Setting**. The Realm is a set of users, credentials, roles, and groups. A user belongs to and logs into a Realm. 
+
+B. **Create Roles**\
+You must select the roles available to users and/or groups. Create new roles with varying degrees of access for the needs of your application. You can also use the default roles by clicking on Client Roles and choosing entandodemo-customerportal-server. The following define their parameters for access:
+* cp-customer -  assigned directly to specific projects for a single customer
+* cp-partner - assigned directly to specific projects for multiple customers
+* cp-support -  read only view of all customer projects 
+* cp-admin - admin access for Customer Portal.
+
+C. **Create New Users**:
+1. The user must be created in Jira first, then configured here for access to the Portal.
+2. From the navigation bar, go to Users. Click `Add User` at right.
+3. Enter the fields as needed but note the requirements for the following:\
+       `Username`: A unique name\
+       `Email`: must use the same address used in Jira\
+       `User Enabled` → On\
+       `Email Verified` → Off\
+    Save
+
+4. From the headings, go to `Credentials`. \
+Under `Credential Reset`, in the `Reset Actions` → Update Password. \
+For external users, you can extend the `Expires In` setting to a minimum of 2 days. \
+Click `Send Email`. \
+This will send an email to the user to activate their account and set a new password.
+5. Role Mappings 
+For each user, select the appropriate `Realm Roles` from the `Available Roles` and click `Add Selected` to assign. If you want to use the Customer Portal default roles, choose `entandodemo-customerportal-server` from the `Client Roles` pull-down menu. This will give you additional roles that you can add. Check the Effective Roles column to make sure the correct roles have been assigned.
+
+![Entando ID Management Role Mapping](./images/custportal-identity-userrole.png)
+
+7. Under Groups, assign to appropriate groups. Roles are additive as in RBAC models.
+
+### Customer Portal Administrator
+
+As administrator for the Customer Portal, you can create and manage customers, projects and subscriptions. You will also be able to assign projects to users who have activated their account on Jira  and have been assigned roles in the Entando Identity Management System. 
+
+The administrator also needs a Jira account and the correct role assignment in the Entando ID management system.
```

Comment:
would lead paragraph with this - has to happen before anything else

Reply:
I think this depends on how people read these sections, and in my mind, they are looking for operations/functions first?